rancher/distros-test-framework
# Add support for kine in rke2

#### Proposed Changes ####


Adding support for the new kine feature implemented in rke2


- No, Need to update JJB to add the new external datastore variables

#### Types of Changes ####


- New functionality to add external datastore in rke2

#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
- Ok

2. Verify code lint; we should not have errors.
- Ok

3. Update the documentation if needed.
- Ok

4. Update makefile and docker run if adding new tests.
- N/A

5. Run your tests at least 4 times with all configurations needed and possible.
- Ok

6. If needed test with different os types.
- TBA, also run with different CNI's and DB types

#### Linked Issues ####



#### Further Comments ####
